base/doc/ltnews35.tex

```diff
@@ -178,6 +178,10 @@ \section{Document metadata interface}
 means that existing documents are in no way affected by the work that
 is carried out and is in a wider alpha or beta test phase.
 
+Documentation about the new command and 
+already existing keys are in \file{l3meta.pdf} and \file{documentmetadata-support.pdf} 
+and also in the documentation of the \pkg{pdfmanagement-testphase} package.
+
 \emph{either add info about some already existing keys or add a
   reference to the place the the right documentation about this is
   stored (to be determined).}
```

Comment:
This would need to go, but the documentation in documentmetadata-support.pdf is obsolete as far as I can see

Reply:
yes, it was still there as a reminder to check if the text before is enough. documentmetadata-support.pdf still contained a description of the the activate key, I just removed that, but beside this I think the documentation there is fine.  There are two open comments in the documentmetadata-support dtx: if two definitions should move into file and if should we use `meta` as module name? The original code used `document` which is already registered in l3prefix.